arithmetization/src/main/java/net/consensys/linea/zktracer/types/Bytecodes.java

```diff
@@ -15,22 +15,27 @@
 
 package net.consensys.linea.zktracer.types;
 
-import net.consensys.linea.zktracer.module.constants.GlobalConstants;
+import static net.consensys.linea.zktracer.module.constants.GlobalConstants.LLARGE;
+import static net.consensys.linea.zktracer.types.Utils.rightPadTo;
+
 import org.apache.tuweni.bytes.Bytes;
 
 public class Bytecodes {
 
-  public static Bytes16 readBytes(final Bytes data, final long offset, final int sizeToRead) {
-    if (offset >= data.size()) {
+  public static Bytes16 readBytes(
```

Comment:
this is a fix

Reply:
⭐

Reply:
we weren't doing the left padding before (this comes from the veridise audit, where we didn't use the SBO for one of the mmio inst). I made a mistake last time when I implemented it (and didn't see it because the MMIO constraint weren't plugged)